test/Autofac.Integration.AspNetCore.Multitenant.Test/Autofac.Integration.AspNetCore.Multitenant.Test.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Check out the A.E.DI test project now. It still tests under all the TFMs, but there's NOT conditional references anymore. That fixes a lot of the compatibility problems. A.E.DI always requires M.E.DI 6+ now.

Reply:
Okay nvm, I really was able to remove it from the test-project.